docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -74,18 +75,61 @@
 
 ## Python
 
-This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Ethereum network.
+This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
 
 ```python Kovan
 {% include 'samples/PriceFeeds/PriceConsumerV3.py' %}
 ```
 
+{% endmarkdown %}
+
 <div class="row cl-button-container">
     <div class="col-xs-12 col-md-12">
     <a href="https://repl.it/@alexroan/GetLatestPriceWeb3PY" class="cl-button--ghost python-tracked">Run this Python example</a>
     </div>
 </div>
 
+{% markdown %}
+
+## Golang
+
+This example uses [go-ethereum](https://github.com/ethereum/go-ethereum) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
+In order to interact with a smart contract in go, you need to:
```

Comment:
Nit: capitalize (Go)

Reply:
ok thanks. will be resolved in the next push